pkg/cpi/mockingUtils.go

```diff
@@ -77,6 +79,24 @@ func GetEmptyHTTPResponseBodyAndErrorNil() (*http.Response, error) {
 	return &res, nil
 }
 
+//StoreFileInOsAndErrorNil - Method to store read artifacts in the file system
+func StoreFileInOs(FilePath string, resp *http.Response) error {
```

Comment:
I think this function is in the wrong place. This is used by the productive code, but it is here in the **mocking**Utils.

Reply:
Yes agree. Moved it to commonUtils